src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
for future instead of &__card:hover

```suggestion
    transition-duration: 300ms;
    
    &:hover {
      cursor: pointer;
      color: #34568b;
      transform: scale(1.2);
    }
    
```